FileOperations/FileOperations/FileDescription.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FileOperations
+{
+	class FileDescription
+	{
+        public const string Path = "C:/Users/divya.singh/Desktop/demo/testFile.txt";
+
+		public void createFileAndAddText()
+		{
+			String line;
+			try
+			{
+				//Passing the file path and file name to the StreamReader constructor
+				StreamReader sr = new StreamReader(Path);
+
+				//Read the first line of text
+				line = sr.ReadLine();
+
+                //Continue to read until you reach end of file
+                while (line != null)
+				{
+					//write the line to console window
+					Console.WriteLine(line);
+					//Read the next line
+					line = sr.ReadLine();
+				}
+
+				//close the file
+				sr.Close();
+				Console.ReadLine();
+			}
+			catch (Exception e)
+			{
+				Console.WriteLine("Exception: " + e.Message);
+			}
+			finally
+			{
+				Console.WriteLine("Executing finally block.");
+			}
+		}
+
+		public void appendFile()
+		{
+			// Adding this text in the file
+			try
+			{
+				using (StreamWriter sw = File.AppendText(Path))
+				{
+					sw.WriteLine("This");
+					sw.WriteLine("is Extra");
+					sw.WriteLine("Text");
```

Comment:
Better approach would be to append a single line of text in a new line.

Reply:
I have made changes